src/getTime.js

```diff
@@ -18,7 +18,8 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  let timeFromString = /\b(2[0-3]|[01]\d):[0-5]\d\b/.exec(str);
```

Comment:
this is an array, so maybe timesArr, timesFromString

Reply:
done :-) Thank you for the remark. Don't know why did I put them ...